internal/core/data/init.go

```diff
@@ -45,7 +45,12 @@ var msc metadata.DeviceServiceClient
 var httpErrorHandler errorconcept.ErrorHandler
 
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the data service.
-func BootstrapHandler(wg *sync.WaitGroup, ctx context.Context, startupTimer startup.Timer, dic *di.Container) bool {
```

Comment:
The function signature already fits on a single line (within the 120-word limit), should you wish to collapse the lines. This comment applies elsewhere, too.